Makefile

```
@@ -21,7 +21,7 @@ build: build-bin build-docker ## All-in-one build
 .PHONY: build-bin
 build-bin: export CGO_ENABLED = 0
 build-bin: fmt vet ## Build binary
-	@go build -o $(BIN_FILENAME) ./...
+	@go build -o $(BIN_FILENAME) github.com/vshn/cloudscale-metrics-collector
```

I think you could just fix it with this:
```suggestion
	@go build -o $(BIN_FILENAME) .
```

---

Indeed it appears to work. Thanks!